src/name_bazaar/ui/events/public_resolver_events.cljs

```diff
@@ -47,3 +47,56 @@
                         :public-resolver.record/addr] (if (= addr "0x")
                                                         d0x-shared-utils/zero-address
                                                         addr)))}))
+(reg-event-fx
+ :public-resolver.record-hash/load
```

Comment:
As I described on Slack earlier, this step is not really needed. We can calculate hash on client side as `(namehash "<address>.addr.reverse")`